crates/btc-relay/rpc/src/lib.rs

```diff
@@ -1,24 +1,34 @@
 //! RPC interface for the BtcRelay Module.
 
 use codec::Codec;
-use jsonrpc_core::{Error as RpcError, ErrorCode, Result as JsonRpcResult};
-use jsonrpc_derive::rpc;
+use jsonrpsee::{
+    core::{async_trait, Error as JsonRpseeError, RpcResult},
+    proc_macros::rpc,
+    types::error::{CallError, ErrorCode, ErrorObject},
+};
 use sp_api::ProvideRuntimeApi;
 use sp_blockchain::HeaderBackend;
 use sp_runtime::{generic::BlockId, traits::Block as BlockT, DispatchError};
 use std::sync::Arc;
 
-pub use self::gen_client::Client as RefundClient;
 pub use module_btc_relay_rpc_runtime_api::BtcRelayApi as BtcRelayRuntimeApi;
 
-#[rpc]
+#[rpc(client, server)]
```

Comment:
Can we make use of the client part in subxt?

Reply:
Hopefully, but I haven't tested this yet.